Automattic/wp-calypso
# Jetpack Connect: add Jetpack footer to login form

Fixes #17571

- Move footer into its own component.
- Remove deprecated PropTypes

**To test**

You must be logged out of your WordPress.com account.

0. Check out this branch. 
1. install/activate Jetpack on a self-hosted site. Make sure you use an email address that's not already associated with a WP.com account, to avoid any matching
2. Copy the link appearing on the blue connect button under the Jetpack menu in wp-admin
3. Add `&calypso_env=development` to the end of that URL.
4. Load the URL. You will land on a page like below, which should include the Jetpack footer:

<img width="1214" alt="screenshot 2017-09-14 at 16 12 52" src="https://user-images.githubusercontent.com/426388/30459796-a1360bb0-9967-11e7-83b0-cd5ccfea82d8.png">

To make sure the oAuth login page still works and does not include the Jetpack footer, try the following:

1. Go to https://stackedit.io/editor#
2. In the left sidebar, click on Publish > WordPress
3. Enter a Jetpack site URL and start the flow.
4. Copy the URL appearing in the pop up to connect to WordPress.com.
5. Replace wordpress.com by `calypso.localhost:3000`
6. Make sure that the alternate footer  with links appears.
